koku/koku/settings.py

```diff
@@ -189,7 +189,7 @@
     os.path.join(BASE_DIR, '..', 'apidoc'),
 ]
 
-STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+# STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
This was causing problems with `DJANGO_DEBUG = False`.  This looks like its used to store .css files or some files needed to serve web page content.  Open to suggestions on how we should handle this.

Reply:
From http://whitenoise.evans.io/en/stable/django.html#make-sure-staticfiles-is-configured-correctly

> As part of deploying your application you’ll need to run ./manage.py collectstatic to put all your static files into STATIC_ROOT. 

Basically we aren't putting our static files where we're telling the thing to check for static files so it 💥 s. So I think if debug is false we just need a flow that is closer to deploying to production in which we run that first.

Reply:
Thats it, thanks!  I'll revert this